inc/package.class.php

```diff
@@ -148,10 +148,13 @@ protected static function getMaxFileSize() {
       switch ($last) {
          case 'g':
             $val *= 1024;
+            break;
          case 'm':
             $val *= 1024;
+            break;
          case 'k':
             $val *= 1024;
+            break;
```

Comment:
and here

If we have 256g as argument 256 will be multiplied 3 times by 1024 to be converted into bytes.

Reply:
I have the doubt about this because my IDE show this alert "Case clause execution can unintentionally falls through the next case, which is most often an error. A break or a return statement at the end of the case will prevent this error."